en/04_Changelogs/5.0.0.md

```diff
@@ -30,98 +30,1020 @@ title: 5.0.0 (unreleased)
 
 This is a major release and as a result there are a number of breaking API changes. For a full list of these see [upgrading your project](/upgrading/upgrading_your_project). Some specific details about a few of them are below.
 
-### Removed API (by module, alphabetically) {#api-removed}
+### Removed and changed API (by module, alphabetically) {#api-removed-and-changed}
 
 #### dnadesign/silverstripe-elemental
 
-- Removed deprecated class `DNADesign\Elemental\Search\ElementalSolrIndex`.
-- Removed deprecated class `DNADesign\Elemental\TopPage\TestState`.
```

Comment:
Why are we now saying these weren't removed? They definitely were. Does your tool not notice if a whole class has been removed?

Reply:
Ah yup sorry, I had a small bug that meant deprecated classes weren't showing, have fixed